cord/client.py

```diff
@@ -147,9 +147,7 @@ def __getattr__(self, name):
                     message=('{} is implemented in Datasets, not Projects.'
                              .format(name))
                 )
-            elif name == 'items':
-                pass
-            else:
+            elif name != 'items':
```

Comment:
Function `CordClient.__getattr__` refactored with the following changes:

- Remove empty elif clause ([remove-pass-elif](https://docs.sourcery.ai/refactorings/remove-pass-elif/))